buildSrc/subprojects/plugins/plugins.gradle.kts

```
@@ -11,6 +11,7 @@ dependencies {
     implementation("org.ow2.asm:asm:7.1")
     implementation("org.ow2.asm:asm-commons:7.1")
     implementation("com.google.code.gson:gson:2.7")
+    implementation("org.gradle:test-retry-gradle-plugin:1.1.0")
```

```suggestion
    implementation("org.gradle:test-retry-gradle-plugin:1.1.1")
```

Can you use the newer version, which fixes a bug which we also may hit?